src/examples/poi-crosschecker/poi-crosschecker.ts

```diff
@@ -100,20 +108,19 @@ const run = async () => {
         `Payload: Subgraph (ipfs hash): ${subgraph}\nnPOI: ${nPOI}\n`.green
       );
 
-      const attestation: Attestation = {
-        nPOI: nPOI,
-        deployment: subgraph,
-        blockNumber: Number(blockNumber),
-        indexerAddress,
-        stakeWeight: Number(stakeWeight),
-      };
-
-      storeAttestations(nPOIs, attestation);
-      return nPOIs;
-    } catch (error) {
-      console.error(`Failed to handle a message into attestation, moving on`, {
-        error: error.message,
+      db.serialize(() => {
+        const stmt = db.prepare("INSERT INTO npois VALUES (?, ?, ?, ?, ?)");
+        stmt.run(
+          subgraph,
+          blockNumber,
+          nPOI,
+          sender,
+          sender === operatorAddress ? 0 : stakeWeight
```

Comment:
I wonder if we should keep sender's own indexer's stakeWeight? Given relationship between sender and indexer, we could assume that sender trust indexer and that local nPOI might be valuable during attestations?

Reply:
This is a very good point! I think it makes sense to zero-our our own attestation, because essentially we want to check cases where we are the "wrong" party. By "we" I mean all three - sender, operator and indexer, since they are essentially the same, given a 1 to 1 mapping of indexer address and operator address

Reply:
that makes sense! I'm thinking of a very simple situation, that do we assume the majority of all stake is honest, or do we assume that the majority of (all stake - our stake) is honest?